docs/examples/data/torchvision/data.py

```diff
@@ -0,0 +1,12 @@
+"""Make sure the data is available"""
+import sys
+import time
+
+from torchvision.datasets import INaturalist
+
```

Comment:
I suggest moving all the moving / copying / symlinking / etc from `data.sh` to this `data.py` file.
This would ideally use `pathlib.Path`s and `shutil` to do stuff.